### PR TITLE
feat: Add a /environment endpoint

### PR DESCRIPTION
**Pull request by larsks**
Provide a handy way to inspect environment variables. Good for verifying
behavior of env/envFrom in your pod manifests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,10 @@ The unit of measure, if specified, accepts the following values: `KB`, `MB`, `GB
 
 WebSocket echo.
 
+### `/environment`
```
Comment (ldez) [inline README.md:37]:
```suggestion
### `/env`
```

File: app.go [hunk 2]
```diff
@@ -64,6 +64,8 @@ func main() {
 	mux.Handle("/bench", handle(benchHandler, verbose))
 	mux.Handle("/api", handle(apiHandler, verbose))
 	mux.Handle("/health", handle(healthHandler, verbose))
+	mux.Handle("/environment", handle(environmentHandler, verbose))
+	mux.Handle("/env", handle(environmentHandler, verbose))
```
Comment (ldez) [inline app.go:68]:
```suggestion
	mux.Handle("/env", handle(environmentHandler, verbose))
```

**Review by ldez — CHANGES_REQUESTED** [review]

File: app.go (hunk 2 above)
Comment (larsks) [reply app.go]: If you don't want to allow both I'd vote for `/environment`, because no other endpoints are abbreviated.

Comment (ldez) [reply app.go]: > because no other endpoints are abbreviated.

We already have `/bench` instead of `/benchmark`

**Comment by ldez** [comment]
I added the information in a different way: a query parameter on `/` and `/api`.

932937c4e8307a4ed8b5152bd871022746453de6

**Pull request closed without merging**